utils/udp.go

```diff
@@ -73,7 +73,7 @@ func NewUDPReceiver(cfg *UDPReceiverConfig) (*UDPReceiver, error) {
 		}
 
 		if cfg.Workers <= 0 {
-			cfg.Workers = cfg.Sockets
+			cfg.Workers = cfg.Sockets * 2
```

Comment:
I'm hesitating making this the default inside the library since this can be configured separately.

Reply:
I thought so, so I will not insist :)